### ISSUE TITLE
Sync from Google to Outlook not Being marked as Private

### ISSUE THREAD

**Issue by TheCrick** [phw198/OutlookGoogleCalendarSync#1372]
> Checklist:-
> * [X] [Upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Upgrading) to the latest release (alpha if available) ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg)
> * [X] Reproduce the issue
> * [X] Complete the information below
> * [X] Provide your `OGcalsync.log` file (see [how-to](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file))

**OGCS Version**: v2.9.2.0  
**Edition**: Installed  
**Sync Direction**:  O<->G

**Bug Description**  
I have two way sync on with private set on. My events from Outlook to Google are set to private but not the other way around. 

Meaning evenings from google calendar are not synced as private to outlook. 

**Steps to Reproduce the Issue**  
Tried on both 2.9 and 2.9.2

**Any other information**  
Any additional information that may help troubleshoot the issue.  

:paperclip: Attach your `OGcalsync.log` file.
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/7910679/OGcalsync.log)

**Comment by phw198** [comment]
Yes, what you have described would be as expected - you have configured OGCS to set Google items as private if synced from Outlook and it is doing that. If it then synced all those items _back_ to Outlook as private that would be a Bad Thing - everything in Outlook would become private!

However, if you created a new Google item as private, OGCS should sync that as private into Outlook.
